src/functions.stub.php

```diff
@@ -26,6 +26,16 @@ function toPHP(string $bson, ?array $typemap = null) {}
     function toRelaxedExtendedJSON(string $bson): string {}
 }
 
+namespace MongoDB\Driver\Logging {
+    function addLogger(Logger $logger): void {}
```

Comment:
Realistically, this is also going to need an optional integer that specifies the minimum logging level so that we can default that to `MONGOC_LOG_LEVEL_DEBUG` and suppress trace-level logs by default. This will also allow us to preserve the optimization whereby PHPC disables trace logging unless absolutely necessary (previously only if `mongodb.debug` was active).

Reply:
No longer needed because we decided to suppress trace-level logs from registered Logger instances.